scripts/ocr_evaluation.py

```diff
@@ -1,104 +1,124 @@
 import io
+import json
 import matplotlib.patches as patches
 import matplotlib.pyplot as plt
 import os
 import requests
 
 from PIL import Image
 
-from supercontrast.client import supercontrast_client
-from supercontrast.provider import Provider
-from supercontrast.task import OCRRequest, OCRResponse, Task
-
-# Define providers
+from supercontrast import (
+    OCRRequest,
+    OCRResponse,
+    Provider,
+    SuperContrastClient,
+    Task,
+    TaskMetadata,
+)
+
+# Constants
+TEST_IMAGE_URLS = [
+    "https://jeroen.github.io/images/testocr.png",
```

Comment:
we should host our own mock data rather than pulling from these goobers

Reply:
True, will fix later